lib/omniauth/strategies/drip.rb

```diff
@@ -17,7 +17,7 @@ class Drip < OmniAuth::Strategies::OAuth2
         :token_url => 'https://www.getdrip.com/oauth/token'
       }
 
-      uid { access_token.client.id }
+      uid { raw_info['accounts'][0]['id'] }
```

Comment:
I think this should actually be a unique identifier corresponding to the _user_. For Drip, the user's email address is this unique identifier.

```
uid { user_info["users"][0]["email"] }
```

For example, it's possible that a user does not actually have access to any accounts.